app/src/main/java/org/systers/mentorship/view/adapters/AchievementsAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package org.systers.mentorship.view.adapters
+
+import android.support.v7.recyclerview.extensions.ListAdapter
+import android.support.v7.util.DiffUtil
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.achievement_list_item.view.*
+import org.systers.mentorship.R
+import org.systers.mentorship.models.Task
+
+/**
+ * This class is the recyclerview adapter for achievements. It is a subclass of [ListAdapter] for
+ * easy async calculation of diffs using DiffUtil to provide nice animations when the data set
+ * changes.
+ */
+class AchievementsAdapter : ListAdapter<Task, AchievementsAdapter.ViewHolder>(AchievementsItemCallback()) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): AchievementsAdapter.ViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.achievement_list_item, parent, false)
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: AchievementsAdapter.ViewHolder, position: Int) {
+        holder.bind(getItem(position))
+    }
+
+    /**
+     * This class holds a view for each item of the Achievements list
+     * @param itemView represents each view of achievements list
+     */
+    inner class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+        /**
+         * This function binds the description of the achievement to the textview
+         * @param task The Achievement whose description is to be bound
+         */
+        fun bind(task: Task) {
+            itemView.tvDescription.text = task.description
+        }
+    }
+
+}
+
+/**
+ * This class represents the DiffCallback for tasks, and is used in [AchievementsAdapter]
+ */
+class AchievementsItemCallback : DiffUtil.ItemCallback<Task>() {
```

Comment:
This callback enables the use of DiffUtil on the RecyclerView for Achievements on the home screen. Using this callback we can get smooth animations when the underlying dataset of the RecyclerView Adapter changes. It's not necessary to use DiffUtil with any RecyclerView, but I thought it would be nice to have it. It does improve the peformance though, because it is more efficient than calling `notifyDatasetChanged()`.